lib/stream_data.ex

```diff
@@ -2094,6 +2094,85 @@ defmodule StreamData do
     end)
   end
 
+  @doc """
+  Generates arbitrary (past and future) dates.
+
+  Generated values shrink towards `Date.utc_today/0`.
+  """
+  @spec date() :: t(Date.t())
+  def date do
+    date([])
+  end
+
+  @doc """
+  Generates dates according to the given `options` or `date_range`.
+
+  ## Options
+
+  * `:origin` - (`Date`) if present, generated values will shrink towards this date. Cannot be combined with `:min` or `:max`.
```

Comment:
Let's replace types with links, here it would be:


```suggestion
  * `:origin` - (`t:Date.t/0`) if present, generated values will shrink towards this date. Cannot be combined with `:min` or `:max`.
```

Reply:
Makes sense.